### PR TITLE
Replace bsa fancybar with Random Hello Bar

### PR DESCRIPTION
**Pull request by BradDenver**
- hello bar content served from SP Forums Random Hello Bar plugin on sitepoint.com

### CODE REVIEW

**Comment by santouras** [comment]
:+1:

File: assets/javascripts/discourse/initializers/random-hello-bar.js.es6 [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+
+      var EXPIRE_DAYS = 14;
+
+      var RandomHelloBar = function() {
+
+        var isEnabled = true;
+
+        // if jQuery Cookie is available, we may have disabled ourselves for the number of days set in EXPIRE_DAYS if user manually closed the bar
+        if ($.cookie) {
+          if ($.cookie('awesomeBarDisabled')) {
```
Comment (tancnle) [inline assets/javascripts/discourse/initializers/random-hello-bar.js.es6]: Could we do `if ($.cookie && $.cookie('awesomeBarDisabled'))`?

Comment (BradDenver) [inline assets/javascripts/discourse/initializers/random-hello-bar.js.es6]: yeah we could. The whole thing will be refactored shortly to remove jQuery all together

**Comment by kellec** [comment]
:+1: as long as we refactor soon

**Comment by daveslutzkin** [comment]
I agree with @kellec.

**Comment by BradDenver** [comment]
Merging coz Dave says its cool too :)

**Pull request merged**